docs/core/tools/using-ci-with-cli.md

```diff
@@ -28,7 +28,7 @@ The installer script is automated to run at the start of the build to fetch and
 > [!NOTE]
 > **Azure DevOps Services**
 >
-> When using the installer script, native dependencies aren't installed automatically. You must install the native dependencies if the operating system doesn't have them. For more information, see [.NET Core dependencies and requirements](../install/dependencies.md?tabs=netcore30&pivots=os-linux).
+> When using the installer script, native dependencies aren't installed automatically. You must install the native dependencies if the operating system doesn't have them. For more information, see [.NET Core dependencies and requirements](../install/dependencies.md?pivots=os-linux).
```

Comment:
There are scripts for both windows and linux. I think we should remove the specific pivot and rely on any previously chosen pivot.

```suggestion
> When using the installer script, native dependencies aren't installed automatically. You must install the native dependencies if the operating system doesn't have them. For more information, see [.NET Core dependencies and requirements](../install/dependencies.md).
```